internal/connector/test/integration/features/connector-multitenancy-api.feature

```diff
@@ -0,0 +1,721 @@
+Feature: connector namespaces API
+  In order to deploy connectors to an addon OSD cluster
+  As a regular user, I need to be able to create and manage namespaces in both evaluation and non-evaluation clusters
```

Comment:
is this sentence correct ? 
I would expect that only the  connector cluster owner/admin can create namespaces on a cluster at this stage.

Reply:
Any user is allowed to create an eval namespace, it automatically gets an expiration time. That's the opt-in flow described in the eval brief document.

Reply:
yeah but the wording here seems to suggest that a user can create a non eval namespace  
do we have a test that verify that only the owner of a cluster can create non eval namespaces ?

Reply:
Right, that's part of the access control requirements I'm going to sort out next after this PR, which will require adding checks for org admin claim.

Reply:
This has been addresed by recent commits to fix authz checks for clusters and namespaces. Only org admins are allowed group wide create and delete access.